src/Service.Tests/SqlTests/RestApiTests/Insert/MsSqlInsertApiTests.cs

```diff
@@ -261,6 +273,68 @@ public async Task TestCleanup()
 
         #endregion
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a rowversion field is missing from the request body.
+        /// In such a case, we skip inserting the field. 
+        /// </summary>
+        [TestMethod]
+        public async Task InsertOneWithRowversionFieldMissingFromRequestBody()
+        {
+            // Validate successful execution of a POST request when a rowversion field (here 'row_version')
+            // is missing from the request body. Successful execution of the POST request confirms that we did not
+            // attempt to provide a value for the 'row_version' field.
```

Comment:
should expand on this comment: "if `row_version` is correctly excluded from the request body, and dab internals attempt to provide a value for the `row_version` field, X error would occur"